README.md

```diff
@@ -118,6 +118,8 @@ Click on the `Grader Console` tab and follow the steps available within the nbgr
 
 * **Workspaces**: User servers are set and launched based on either the user's LTI compatible role (student/learner group or instructor group) or by specifying the ?next=/user-redirect/<workspace_type> as a query parameter that identifies the workspace type by path, for example: next=/user-redirect/theia for the Theia IDE or next=/user-redirect/vscode for VS Code IDE.
 
+* **Shared drive**: An shared folder can be used to share files from the share grader service and the student notebooks.
```

Comment:
A shared folder that may be used to share content among users within a course. Users with access to the course's shared grader notebook (Instructors and TAs) have read/write access to the files located in the shared folder. Users accessing their own workspaces have access to the files in the shared grader notebook with read-only access.